feat(docs): Appointment letter (#655)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md

<!-- add-breadcrumbs -->

# Appointment Letter

> 

Reviewer: ```suggestion
> Introduced in Version 13
```

**The letter written by the employer requesting the selected candidates to join in a specific position.**

In ERPNext you can create an Appointment Letter that you can give to candidates. To access Appointment Letter, go to:

> Home > Human Resource > Recruitment > Appointment Letter

## 1. Prerequisites

Before creating an Appointment Letter, it is advisable to create the following:

* [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)

## 2. How to Create an Appointment Letter
1. Go to an Appointment Letter list, click on New.
1. Select the Job Applicant and Appointment Date.

> 1. You can manually fill Introduction, Terms and, Closing Statement or select an Appointment Letter Template to autofill the content.

Reviewer: ```suggestion
1. You can manually fill Introduction, Terms and, Closing Statement or select an Appointment Letter Template to autofill the content. You can create new Appointment Letter Templates to select them easily.
```

> 1. Click on Save.

Reviewer: ```suggestion
1. Click on Save.
1. After saving, go to the Print View to view and save the PDF of the Appointment letter.
```

    <img class="screenshot" alt="Appointment Letter" src="{{docs_base_url}}/assets/img/human-resources/appointment-letter.png">

> **Note:** Appointment Letter can be made only against a [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)

> ## 2. Features

Reviewer: This should be 3. not 2.

### 2.1 Appointment Letter Template

> To create an Appointment Letter Template:

Reviewer: ```suggestion
```

1. Go to Appointment Letter Template list, click to New.

> 1. fill Introduction, Terms and, Closing Statement.

Reviewer: ```suggestion
1. Fill Introduction, Terms and, Closing Statement.
```

1. Click on Save.

    <img class="screenshot" alt="Appointment Letter" src="{{docs_base_url}}/assets/img/human-resources/appointment-letter-template.png">

> > **Note** On selecting Appointment Letter Template in Appointment Letter it autofills the content.

Reviewer: ```suggestion
> **Note** On selecting Appointment Letter Template in Appointment Letter, it autofills the content.
```

### 2.2 Print Format

> You can create or use existing standard print format.

Reviewer: ```suggestion
You can create new or use existing standard print formats.
```
